pkg/controllers/uiplugin/distributed_tracing.go

```diff
@@ -29,13 +30,36 @@ func createDistributedTracingPluginInfo(plugin *uiv1alpha1.UIPlugin, namespace,
 		extraArgs = append(extraArgs, fmt.Sprintf("-features=%s", strings.Join(features, ",")))
 	}
 
+	proxyName, proxyNamespace := "distributed-tracing-console-plugin", "openshift-tracing"
+
+	if plugin.Spec.TroubleshootingPanel != nil {
+		if plugin.Spec.TroubleshootingPanel.Korrel8r.Name != "" {
+			proxyName = plugin.Spec.TroubleshootingPanel.Korrel8r.Name
+		}
+		if plugin.Spec.TroubleshootingPanel.Korrel8r.Namespace != "" {
+			proxyNamespace = plugin.Spec.TroubleshootingPanel.Korrel8r.Namespace
+		}
+	}
```

Comment:
The code here is for korrel8r and the troubleshooting panel. But I think the intention here is to define this fields in the `DistributedTracingConfig` spec so they can be configured if not using the defaults